book/src/super-sql/declarations/functions.md

```diff
@@ -25,21 +25,21 @@ fn add(a,b): a+b
 ```
 when invoked as
 ```
-values {x:1} | values add(x,1)
+values {x:1} | values add(x,2)
 ```
 is passed the record the `{a:x,b:1}`, which after resolving `x` to `1`,
-is `{a:1,b:1}` and thus evaluates the expression
+is `{a:1,b:2}` and thus evaluates the expression
 ```
 this.a + this.b
 ```
-which results in `2`.
+which results in `3`.
 
-Any function-as-value arguments passed to a function do not appear in the `this`
+Any [function references](../expressions/functions.md#function-references) passed to a function do not appear in the `this`
```

Comment:
I assume this is an appropriate substitution (wanted something we could link to rather than invoking a new term), but please double check.